### PR TITLE
{tools}[system] rclone v1.42

### PR DESCRIPTION
**Pull request by JackPerdue**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
sms - Linux centos linux 7.6.1810, AMD EPYC 7601 32-Core Processor, Python 2.7.5
See https://gist.github.com/36adf1138a2e95317cc29bcdeab385de for a full test report.

**Comment by migueldiascosta** [comment]
@JackPerdue, this PR is being closed for the following reason(s): closing and reopening to trigger tests.

**Comment by JackPerdue** [comment]
Seems rather anal/silly to fail because I have http:// (not https://) IN A COMMENT.

File: easybuild/easyconfigs/r/rclone/rclone-1.42-amd64.eb [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+ a variety of online storage services
+"""
+
+toolchain = {'name': 'system', 'version': ''}
+
+source_urls = ['https://downloads.%(name)s.org/v%(version)s/']
+sources = ['%(name)s-v%(version)s-linux%(versionsuffix)s.zip']
+checksums = ['7a623f60a5995f33cca3ed285210d8701c830f6f34d4dc50d74d75edd6a5bfa6']
+
+cmds_map = [('', '')]
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/r/rclone/rclone-1.42-amd64.eb]: hm... `CmdCp` derives from `MakeCp` and adds `cmds_map`, so maybe it makes more sense to use `MakeCp` for this? Although I suppose you'd have to skip the `build` step...

Comment (boegel) [reply easybuild/easyconfigs/r/rclone/rclone-1.42-amd64.eb]: Agreed, this is a bit hackish...

**Comment by JackPerdue** [comment]
(damn Europeans with their toilets that shoot agua up their arse)... done  

:)

**Comment by JackPerdue** [comment]
Any updates... I think I made requested changes.

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/41cd411710e9b6577c5711837d226fb4 for a full test report.

**Comment by migueldiascosta** [comment]
Going in, thanks @JackPerdue!

**Pull request merged**